subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_coap

```diff
@@ -5,8 +5,8 @@
 
 menuconfig NRF_CLOUD_COAP
 	bool "nRF Cloud COAP"
+	depends on ZCBOR
```

Comment:
in the `nrf_cloud_multi_service` sample please add `CONFIG_ZCBOR=y` to the .conf files that set `CONFIG_NRF_CLOUD_COAP=y`.

Reply:
Is there some condition in Kconfig or DTS that can let us know that NRF_CLOAD_COAP is enabled? For instance we can add some kconfig fragment so that ZCBOR will have `default y` if one of those conditions are met.

```
if NRF_CLOUD_COAP

config ZCBOR
  default y

endif
```

Reply:
Maybe so, but I'd rather not have ZCBOR intelligently enable itself based on the presence of a downstream dependent library.

I'd rather just have users of `NRF_CLOUD_COAP` enable ZCBOR now that it is no longer automatically selected by the library